src/data/TableData.js

```diff
@@ -0,0 +1,242 @@
+export const judges = [
```

Comment:
can you create a mock folder under data and store any mock data since most of these files is mock data